Gruntfile.js

```diff
@@ -32,24 +32,27 @@ module.exports = function( grunt ) {
 
 		browsers.desktop = [
 			"bs_chrome-45", // shares V8 with Node.js 4 LTS
-			"bs_chrome-60", "bs_chrome-61",
+			"bs_chrome-69", "bs_chrome-70",
 
-			"bs_firefox-45", "bs_firefox-52", // Firefox ESR
-			"bs_firefox-55", "bs_firefox-56",
+			"bs_firefox-52", "bs_firefox-60", // Firefox ESR
+			"bs_firefox-61", "bs_firefox-62",
 
-			"bs_edge-15", "bs_edge-16",
+			"bs_edge-16", "bs_edge-17",
 
 			"bs_ie-9", "bs_ie-10", "bs_ie-11",
 
-			"bs_opera-47", "bs_opera-48",
+			"bs_opera-55", "bs_opera-56",
 
 			// Real Safari 6.1 and 7.0 are not available
-			"bs_safari-6.0", "bs_safari-8.0", "bs_safari-9.1", "bs_safari-10.1", "bs_safari-11.0"
+			"bs_safari-6.0", "bs_safari-8.0", "bs_safari-9.1", "bs_safari-10.1",
+			"bs_safari-11.1", "bs_safari-12.0",
 		];
 
 		browsers.ios = [
-			"bs_ios-5.1", "bs_ios-6.0", "bs_ios-7.0", "bs_ios-8.3",
-			"bs_ios-9.3", "bs_ios-10.3"
+			"bs_ios-5.1", "bs_ios-6.0", "bs_ios-7.0", "bs_ios-8.3", "bs_ios-9.3", "bs_ios-10.3",
+
+			// iOS 11.* & 12.0 launchers don't work so they have to be skipped for now.
```

Comment:
oh :/

Reply:
I already contacted BrowserStack about that. They don't work in different ways.